src/app/clusters/webrtc-transport-provider-server/webrtc-transport-provider-server.cpp

```diff
@@ -271,29 +271,67 @@ void WebRTCTransportProviderServer::HandleSolicitOffer(HandlerContext & ctx, con
         return;
     }
 
-    // Validate VideoStreamID against AllocatedVideoStreams if present and not null
-    if (req.videoStreamID.HasValue() && !req.videoStreamID.Value().IsNull())
+    // At least one of Video Stream ID and Audio Stream ID has to be present
+    if (!req.videoStreamID.HasValue() && !req.audioStreamID.HasValue())
     {
```

Comment:
This part has been removed from the spec,  since both of those commands have Group conformance set on those paramaters, so it should be automatically handled per comment, so I removed this check. But there is nothing wrong to add this back